tests/drivers/candev/main.c

```diff
@@ -226,24 +226,32 @@ if (IS_ACTIVE(CONFIG_USE_LOOPBACK_MODE)) {
         struct can_filter filter[4];
```

Comment:
I know this block wasn't changed in this PR, but the indentation appears to be off? 
From what I can see in the Blame, it's from your previous PR, about a year ago. Maybe also fix it in this one?

(Starts two lines higher up than this comment, but Github doesn't let me add a comment there)